nixos/modules/services/hardware/uinput.nix

```diff
@@ -0,0 +1,28 @@
+{ config, pkgs, lib, ... }:
+
+let
+  cfg = config.services.hardware.uinput;
+in {
+  options.services.hardware.uinput = {
+    enable = lib.mkOption {
+      type = lib.types.bool;
+      default = false;
+      description = "Whether to enable uinput support";
+    };
+  };
+
+  config = lib.mkIf cfg.enable {
+    boot.kernelModules = [ "uinput" ];
+
+    users.users.uinput = {
+      group = "uinput";
```

Comment:
I don't think this needs to be a user though. Just the group `uinput` would be better in this module and an `xow` user in the xow module that's part of that group.

Reply:
True. Fixed.